src/main/java/org/isf/therapy/model/TherapyRow.java

```diff
@@ -250,8 +247,7 @@ public void setSms(boolean sms) {
 	}
 
 	public String toString() {
-		String string = medicalId.toString() + " - " + this.unitID + " " + this.qty + "/" + this.freqInDay + "/" + this.freqInPeriod;
-		return string;
+		return medicalId.toString() + " - " + this.unitID + " " + this.qty + "/" + this.freqInDay + "/" + this.freqInPeriod;
```

Comment:
this one could be replaced by a StringBuilder

Reply:
done.